urdf/CMakeLists.txt

```diff
@@ -27,7 +27,7 @@ catkin_package(
   LIBRARIES ${PROJECT_NAME}
   INCLUDE_DIRS include ${TinyXML_INLCLUDE_DIRS} ${CATKIN_DEVEL_PREFIX}/include
```

Comment:
I believe the `TinyXML_INLCLUDE_DIRS` (spelling error???) becomes obsolete with this change.